From 07a335b585c780befa91d425b8d70cf2cd32123a Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@lsi.com>
Date: Fri, 28 Feb 2014 13:48:14 -0600
Subject: [PATCH 48/71] arch/powerpc: Clean up of Compatible Names in Axxia
 Device Trees

git://git.yoctoproject.org/linux-yocto-3.10 standard/axxia/base
commit 5bec52b225ffc4bb27b9bdd35bc555c93d84b2d4

The base compatible type for Axxia (PowerPC) is not lsi,acp.

Signed-off-by: John Jacques <john.jacques@lsi.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/powerpc/platforms/44x/acpx1.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/platforms/44x/acpx1.c b/arch/powerpc/platforms/44x/acpx1.c
index b903787..934ea0a 100644
--- a/arch/powerpc/platforms/44x/acpx1.c
+++ b/arch/powerpc/platforms/44x/acpx1.c
@@ -253,7 +253,7 @@ static int __init acpx14xx_probe(void)
 {
 	unsigned long root = of_get_flat_dt_root();
 
-	if (!of_flat_dt_is_compatible(root, "ibm,acpx1-4xx"))
+	if (!of_flat_dt_is_compatible(root, "lsi,acp"))
 		return 0;
 
 	return 1;
-- 
1.7.5.4

